dropshot/src/server.rs

```diff
@@ -597,9 +643,13 @@ impl<C: ServerContext> InnerHttpsServerStarter<C> {
         });
 
         let make_service = ServerConnectionHandler::new(Arc::clone(&app_state));
-        let server = Server::builder(https_acceptor).serve(make_service);
+        //let server = Server::builder(https_acceptor).serve(make_service);
```

Comment:
we're moving this into InnerHttpsServerStarter::start effectively?

Reply:
Yes, I just removed this line now. But hyper got rid of the `Accept` pattern, so the accept loop is done within this file.